src/query/storages/parquet/src/parquet_rs/statistics/column.rs

```diff
@@ -79,7 +79,17 @@ pub fn convert_column_statistics(s: &Statistics, typ: &TableDataType) -> ColumnS
                     TableDataType::Number(NumberDataType::Int64) => {
                         (Scalar::from(max), Scalar::from(min))
                     }
-                    TableDataType::Timestamp => (Scalar::Timestamp(max), Scalar::Timestamp(min)),
+                    TableDataType::Timestamp => {
+                        let multi = match max.checked_ilog10().unwrap_or_default() + 1 {
```

Comment:
as a statistic of some data, max can be any value from 0? 
not always for `2023...`?

Reply:
> max can be any value start from 0

Yes,  if the number of digits is less or equal  than  10, we set the unit to be second.  This could enlarge the range if the real precision is `ms` or others. But it will still work correctly in range pruner.

> data not always 2023...

Yes, the  timestamp value is time seconds(mills, micro, nanos) from epoch.